src/sass/Fabric.Scoped.scss

```diff
@@ -4,8 +4,12 @@
 // Office UI Fabric
 // Core styles scoped to the current major version of Fabric
 
+// Fabric version gets set on compile
```

Comment:
Could you expand this to say something along the lines of "$FabricVersion becomes a SASS list like [1,2,3] on build"